deps/verifier/Cargo.toml

```
@@ -49,6 +49,7 @@ strum.workspace = true
 veraison-apiclient = { git = "https://github.com/chendave/rust-apiclient", branch = "token", optional = true }
 ear = { git = "https://github.com/veraison/rust-ear", rev = "43f7f480d09ea2ebc03137af8fbcd70fe3df3468", optional = true }
 x509-parser = { version = "0.14.0", optional = true }
+reqwest.workspace = true
```

It is good to mark this dependency to feature `snp-verifier`. s.t.
```suggestion
reqwest = { optional = true, workspace = true }
```

and add `"request"` to previous `[features]` section
```toml
...
[features]
snp-verifier = [ "asn1-rs", "openssl", "sev", "x509-parser", "reqwest" ]
...
```